core/src/test/kotlin/api/RunsRouteIntegrationTest.kt

```diff
@@ -1114,7 +1126,8 @@ class RunsRouteIntegrationTest : AbstractIntegrationTest({
                     JobConfigurations(),
                     "jobConfigContext",
                     labelsMap,
-                    traceId = "t3"
+                    traceId = "t3",
+                    null
```

Comment:
Shouldn't there be at least one occurrence where a value is set for the new property to test the mappings?